src/bronx/compat/moves.py

```diff
@@ -20,15 +18,14 @@ def _require_version(major, minor=0):
 
 
 # ABCs are moved from "collections" to "collections.abc" in 3.8
-six.add_move(six.MovedModule("collections_abc", "collections", "collections.abc"))
-collections_abc = six.moves.collections_abc
+six.add_move(six.MovedModule("collections.abc", "collections", "collections.abc"))
+collections.abc = six.moves.collections_abc
```

Comment:
``six`` should not be used anymore.

``collections_abc = collections.abc``

Should be enough.